tests/plots/test_declarative.py

```diff
@@ -394,6 +394,39 @@ def test_declarative_layers_plot_options():
     return pc.figure
 
 
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=1.12)
```

Comment:
Why does a new test need such a high threshold by default?

Reply:
I think this is because I used some dot-dash and dotted lines to test different linestyles. This caused an issue, probably with the different versions of matplotlib.